contentcuration/contentcuration/viewsets/common.py

```diff
@@ -54,6 +56,12 @@ class SQSum(Subquery):
     output_field = IntegerField()
 
 
+class SQArrayAgg(Subquery):
```

Comment:
Just flagging that if we use this more - we may end up need to make the inner output field conditionalizable. There's also some coersion stuff that Django does not do by default iirc - so we may need to overwrite some more methods to ensure that stuff gets run through the right type conversion in some cases.